gm4_blast_furnaces/data/blast_furnaces/functions/check_ore.mcfunction

```diff
@@ -0,0 +1,10 @@
+# @s = blast furnace that is looking at a lit furnace
+# run from processes
```

Comment:
This is not run from processes. But after my other comment in `verify_furnace", it should just be called from processes.